nixos/modules/hardware/inputmodule.nix

```diff
@@ -0,0 +1,18 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
```suggestion
```

Use `lib.foo` as you don't have a lot of lib functions (You can use `let inherit (lib) foo; in` otherwise)